### ISSUE TITLE
Bookmarked folders icons broken 

### ISSUE THREAD

**Issue by POPCORNS1068** [linuxmint/nemo#1979]
```
 * Nemo version - 4.0.0
 * Is issue with desktop or windowed nemo? Both
 * Distribution - Arch
 * NVIDIA GeForce GTX 1080, 410.66-3
 * 64 bit
 ```

**Issue**
I'm using the Mint-X icon theme, bookmarked folders' icons change to some weird fallback (looks like adwaita, not sure) after accessing them once, and stay like that until Nemo is killed.

**Steps to reproduce**
* Bookmark a folder by dragging it to the sidebar
* Click on the bookmark
* This happens
![image](https://user-images.githubusercontent.com/40209997/48199760-11ebcb80-e398-11e8-9c2e-9b051d3f37fe.png)

**Expected behaviour**
The icon should stay as it was.

**Other information**
XDG user directories are not affected by this.

**Comment by NikoKrause** [comment]
That brown line issue is inherited from Adwaita: https://gitlab.gnome.org/GNOME/adwaita-icon-theme/issues/10

**Comment by POPCORNS1068** [comment]
Ok I just realized some other icon themes like Papirus doesn't have this issue, so that means it can be fixed by messing with the icon theme itself, but I have no idea how this could be done as Mint-X and Papirus's directory structures are pretty different.

I really don't want to switch from Mint-X, what can I do to fix this?
(It doesn't feel like Mint-X's upstream will fix this anytime soon either)

**Comment by NikoKrause** [comment]
The Adwaita icon has been fixed, but I don't think, that the new Adwaita icon package will make it into Ubuntu 18.04.

But there's a fix for Mint-X. The icon shouldn't change if you click on it. It turns out, that Mint-X is missing an icon. I have created a Pull Request, so this will be fixed in Mint 19.1 (about December).

Till the fix gets merged, you can fix it on your own:
```
cd /usr/share/icons/Mint-X/places/scalable/
sudo ln -s folder-symbolic.svg inode-directory-symbolic.svg
sudo gtk-update-icon-cache /usr/share/icons/Mint-X/
```

This will create the missing icon as a symbolic link and updates the icon cache.
